### ISSUE TITLE
Warning during compilation

### ISSUE THREAD

**Issue by Sabryr** [torognes/vsearch#321]
I got this warning when compiling on our CentOS 7 system, using gcc/8.1.0. It installed fine, just letting you know of the warning as not sure whether this will coarse any run time issues.  

  strncpy(*cigarendp, buf, (size_t)len);
       ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function ‘void pushop(char, char**, char*, int*)’,
    inlined from ‘void pushop(char, char**, char*, int*)’,
    inlined from ‘void nw_align(char*, char*, char*, char*, int64_t*, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t, int64_t*, int64_t*, int64_t*, int64_t*, int64_t*, char**, int64_t, int64_t, nwinfo_s*)’ at align.cc:430:11:
align.cc:88:14: warning: ‘char* strncpy(char*, const char*, size_t)’ output may be truncated copying between 1 and 10 bytes from a string of length 24 [-Wstringop-truncation]

**Comment by torognes** [comment]
Thank you for reporting this.

This problem seems very similar to issue #304, which should have been solved in the latest commit (5d6be05a1403eb4725fd9e020fe0aa6ffd29a282), but which is not included in the latest release (2.8.0).

I will release a new version 2.8.1 with the fix.

It should not cause any runtime problems.

**Comment by torognes** [comment]
Fixed in version 2.8.1 just released.
